### PR TITLE
Simplify method

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: osu.Game/Screens/Menu/MainMenu.cs [hunk 1]
```diff
@@ -104,13 +104,10 @@ private void preloadSongSelect()
 
         public void LoadToSolo() => Schedule(onSolo);
 
-        private void onSolo() => this.Push(consumeSongSelect());
-
-        private Screen consumeSongSelect()
+        private void onSolo()
         {
-            var s = songSelect;
+            this.Push(songSelect);
```
Comment (swoolcock) [inline osu.Game/Screens/Menu/MainMenu.cs]: This is not the same logic.  With your change, `songSelect` is nulled *after* the call to `Push`, whereas it was previously before it.

File: osu.Game/Online/API/APIAccess.cs [hunk 2]
```diff
@@ -165,7 +165,7 @@ private void run()
                         }
 
                         // The Success callback event is fired on the main thread, so we should wait for that to run before proceeding.
-                        // Without this, we will end up circulating this Connecting loop multiple times and queueing up many web requests
+                        // Without this, we will end up circulating this Connecting loop multiple times and queuing up many web requests
```
Comment (swoolcock) [inline osu.Game/Online/API/APIAccess.cs]: This is just picking at personal preference with words that have more than one accepted spelling.  Unnecessary.

**Review by swoolcock — CHANGES_REQUESTED** [review]

File: osu.Game/Screens/Menu/MainMenu.cs (hunk 1 above)
Comment (pi1024e) [reply osu.Game/Screens/Menu/MainMenu.cs]: Why is nulling after a problem?

Comment (pi1024e) [reply osu.Game/Screens/Menu/MainMenu.cs]: Does it matter?

File: osu.Game/Screens/Menu/OsuLogo.cs [hunk 3]
```diff
@@ -229,7 +229,7 @@ public OsuLogo()
         }
 
         /// <summary>
-        /// Schedule a new extenral animation. Handled queueing and finishing previous animations in a sane way.
+        /// Schedule a new external animation. Handled queuing and finishing previous animations in a sane way.
```
Comment (iiSaLMaN) [inline osu.Game/Screens/Menu/OsuLogo.cs]: Same here for `queuing`, https://github.com/ppy/osu/pull/5883#discussion_r319149348

**Comment by pi1024e** [comment]
Ok. Fixed

**Comment by pi1024e** [comment]
@swoolcock So I changed it so songSelect is nulled before the push.

**Comment by swoolcock** [comment]
What is the purpose of this pull request?

**Comment by smoogipoo** [comment]
Please PR just the typo changes if you wish. The other code is unnecessary and untested.

**Pull request closed without merging**